lms/djangoapps/courseware/views/views.py

```diff
@@ -111,7 +111,8 @@
 # credit and verified modes.
 REQUIREMENTS_DISPLAY_MODES = CourseMode.CREDIT_MODES + [CourseMode.VERIFIED]
 
-CertData = namedtuple("CertData", ["cert_status", "title", "msg", "download_url", "cert_web_view_url"])
+CertData = namedtuple("CertData", ["cert_status", "title", "msg", "download_url",
+                                   "cert_web_view_url", "past_availability_date"])
```

Comment:
Not a nitpick - you really want something much stronger than "past_availability_date" here.  The `past_availability_date` variable that you pass in is based on the result of the `may_certify()` function, which takes several factors into account.  I'd recommend calling this something like "may_view_certificate".